lib/commands/test.js

```diff
@@ -207,13 +202,7 @@ module.exports = Command.extend({
         }).then(() => this.runTask('Test', testOptions));
       }
 
-      return session.finally(this.rmTmp.bind(this));
+      return session;
     });
-  },
-
-  onInterrupt() {
-    this.rmTmp();
-
-    return this._super.onInterrupt.apply(this, arguments);
-  },
+  }
```

Comment:
```suggestion
  },
```
To make the linter happy